src/cryptography/x509/verification.py

```diff
@@ -0,0 +1,7 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from cryptography.hazmat.bindings._rust import x509 as rust_x509
+
```

Comment:
I think you want an `__all__ = ["Store"]` here

Reply:
152a12a26